Description: Remove bundler dependency during tests
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2015-09-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -1,3 +1,2 @@
-require 'bundler/setup'
 require 'minitest/utils'
 require 'minitest/autorun'
